prow/cmd/build/main.go

```diff
@@ -144,6 +144,9 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("Error building client configs")
 	}
+	if defaultContext == "" {
+		defaultContext = kube.DefaultClusterAlias
```

Comment:
This logic should live in `kube.LoadClusterConfigs()` to make sure it always returns the correct value.

Reply:
@fejta I moved the logic in `kube.LoadClusterConfigs()`.